curp/src/server/curp_node.rs

```diff
@@ -161,6 +162,15 @@ impl<C: 'static + Command, RC: RoleChange + 'static> CurpNode<C, RC> {
         Ok(resp)
     }
 
+    /// Handle `ProposeConfChange` requests
+    #[allow(clippy::todo, clippy::unused_async)] // TODO: implement this
+    pub(super) async fn propose_conf_change(
+        &self,
+        _req: ProposeConfChangeRequest,
+    ) -> Result<ProposeConfChangeResponse, CurpError> {
+        todo!("propose_conf_change")
```

Comment:
Can't `raw_curp::apply_conf_change` be used here yet?

Reply:
Similar to `etcd`, `apply_conf_change` will be called when this log entry is apply.